packages/contracts-bedrock/test/vendor/eas/EAS.t.sol

```diff
@@ -0,0 +1,3080 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.15;
```

Comment:
nit: nowhere else in the codebase does it do `=0.8.15`, this is semantically correct but also equivalent to just `0.8.15` which the rest of the codebase uses